apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -133,27 +136,21 @@ export default {
     }
   },
   computed: {
-    ...mapGetters('Files', ['selectedFiles', 'shareOpen', 'shares', 'sharesError', 'sharesLoading']),
+    ...mapGetters('Files', ['selectedFiles', 'shareOpen', 'shares', 'sharesError', 'sharesLoading', 'davProperties']),
     ...mapState(['user']),
     $_ocCollaborationStatus_autocompletePlacholder () {
       return this.$gettext('Add name(s), email(s) or federation ID\'s')
     },
     owner () {
-      if (this.shares.length === 0) {
-        return {
-          avatar: 'https://picsum.photos/64/64?image=1074',
-          name: this.user.id,
-          displayName: this.user.displayname
-        }
-      }
-      const s = this.shares[0]
-      return {
-        avatar: 'https://picsum.photos/64/64?image=1074',
-        name: s.uid_owner,
-        displayName: s.info.displayname_owner
-      }
-    }
+      this.$client.files.fileInfo(this.selectedFiles[0].path, this.davProperties).then(res => {
+        let ownerId = res['fileInfo']['{http://owncloud.org/ns}owner-id']
```

Comment:
owner-id should already be known at this point - this network call shall not be necessary

Reply:
In the previous approach the owner was loaded together with shares. But in case that the recipient doesn't share the file with someone else, the owner wasn't defined. Or do you mean some other way of loading the owner?

Reply:
All I'm saying is that the PROPFIND which loaded the filelist should load all the dav propd we need. And because of that no additional call is now here needed.

Reply:
Moved to buildFile function in store